docs/books/oauthv2.md

```diff
@@ -34,11 +34,11 @@ https://oauth-staging.deskera.xyz/?client_id=cl_pub_1234&scope=write&state=65921
 
 #### Step 2: Users connect their Deskera account
 
-Once on the Deskera App auth page, the user would be prompted to input his/her Deskera user name and password, select the Organization we would like to connect and asked to authorize the app. The user should be already registered to Deskera Books and have atleast one organisation before coming to this step.
+Once on the Deskera App auth page, the user would be prompted to select the organization he/she would like to connect and asked to authorize the app. The user should be already registered to Deskera Books and have atleast one organisation before coming to this step.
```

Comment:
Once on the Deskera App auth page, the user would be prompted to select the organization to connect to and will be asked to authorize the app. Please note user should be already registered  with Deskera Books and have atleast one organisation before this step.